specification/paths/Services-service_id-ServiceContracts-service_contract_id.json

```diff
@@ -0,0 +1,61 @@
+{
+  "get": {
+    "tags": [
+      "Services"
+    ],
+    "summary": "Get a service contract for a service by id.",
+    "description": "This endpoint retrieves a service contract for the supplied service.",
+    "parameters": [
+      {
+        "in": "path",
+        "name": "service_id",
+        "type": "string",
+        "required": true,
+        "description": "Id of the service."
+      },
+      {
+        "in": "path",
+        "name": "service_contract_id",
+        "type": "string",
+        "required": true,
+        "description": "Id of the service contract."
+      },
+      {
+        "in": "query",
+        "name": "include",
+        "type": "string",
+        "description": "Comma seperated list of relationship names you want to include the data of.<br> The relationships that can be included are carrier-contract, service-groups, service-insurances and service-options."
+      }
+    ],
+    "responses": {
+      "200": {
+        "description": "Retrieved the service contract",
+        "schema": {
+          "type": "object",
+          "required": [
+            "data",
+            "links"
+          ],
+          "additionalProperties": false,
```

Comment:
There will be additional properties when `include` is used in the query. Because we can't define what's in the `included` array yet ([anyOf](https://spacetelescope.github.io/understanding-json-schema/reference/combining.html#anyof) isn't supported in swagger 2.0), I suggest removing this for now.